deploy-queue/Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
-name = "mutexbot"
-version = "0.3.2"
+name = "deploy-queue"
+version = "0.1.0"
```

Comment:
This is blocked by the decision in https://databricks.slack.com/archives/C091Z0ZNTE1/p1758272283560079, we should decide that before we merge this.